package com.savior.chatio.application_service.exceptions;

public class InternalTamperingException extends RuntimeException {
    public InternalTamperingException(String message) {
        super(message);
    }
    
}
